src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectDebuggerProvider.cs

```diff
@@ -68,16 +68,17 @@ public override Task<bool> CanLaunchAsync(DebugLaunchOptions launchOptions)
         /// <summary>
         /// Helper returns the correct debugger engine based on the targeted framework
         /// </summary>
-        public static Guid GetManagedDebugEngineForFramework(string targetFramework)
+        public static Guid GetManagedDebugEngineForFramework(string targetFramework, bool mixedMode)
         {
             // The engine depends on the framework
             if (IsDotNetCoreFramework(targetFramework))
             {
+                if (mixedMode) throw new Exception(VSResources.UnmanagedDebuggingNetCoreNotSupported);
```

Comment:
The most common way that project systems do it is with `MixedNativeAndManagedEngine`, but passing both engines should work as well. So you should be able to just add on the native engine if you want to be consistent.